content/en/docs/Customizing/general_configuration.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "General Configuration"
+linkTitle: "General Configuration"
+weight: 3
+date: 2017-01-05
+description: >
+  Configuring a cOS derivative
+---
+
+
+cOS during installation, reset and upgrade (`cos-installer`, `cos-reset` and `cos-upgrade` respectively) will read a configuration file in order to apply derivative customizations. The configuration files are sourced in precedence order and can be located in the following places:
+
+- `/etc/environment`
+- `/etc/os-release`
+- `/etc/cos/config`
+
+A typical configuration file might look like this:
+
+{{<githubembed repo="rancher-sandbox/cos-toolkit" file="examples/standard/files/etc/cos/config" lang="bash">}}
+
+## Reference
+
+Here you can find the full reference of the variables that can be set in the cos configuration file:
+
+- `VERIFY`: Disable/enable image verification during upgrades ( default: true )
+- `CHANNEL_UPGRADES`: Disable/enable upgrades via release channels instead of container images ( default: true )
+- `UPGRADE_IMAGE`: Default container image used for upgrades ( defaults to system/cos with channel `CHANNEL_UPGRADES` enabled ). See also [upgrades](../upgrades)
+- `RECOVERY_IMAGE`: Default recovery image to use when upgrading the recovery partition ( defaults to `recovery/cos` with channel CHANNEL_UPGRADES enabled )
```

Comment:
umm.. on the embedded example this says different:

```
# Default recovery image to use when upgrading the recovery partition.  ( defaults to system/cos with channel CHANNEL_UPGRADES enabled )
```

Maybe we should fix that to be on sync?

Reply:
Good point, yes, it should be fixed. but now that I'm having a closer look we just set it up as part of cloud-init files: https://github.com/rancher-sandbox/cOS-toolkit/blob/f5cca14061d4a3a2d23b7097a99c5d99f475df7c/packages/cloud-config/oem/02_upgrades.yaml#L51 so it will just default to `UPGRADE_IMAGE` if none is set.

Reply:
I've tried to reword it in a way to encompass that, not sure if it's clear enough :sweat_smile:

Reply:
good enough to me, thanks!